### ISSUE TITLE
http://www.ipol.im/pub/pre/231/ 这个地址的pytorch代码有问题

### ISSUE THREAD

**Issue by 7568** [cszn/FFDNet#31]
`
imgn_train = img_train + noise
# Create input Variables
img_train = Variable(img_train.cuda())
imgn_train = Variable(imgn_train.cuda())
noise = Variable(noise.cuda())
stdn_var = Variable(torch.cuda.FloatTensor(stdn))

# Evaluate model and optimize it
out_train = model(imgn_train, stdn_var)
loss = criterion(out_train, noise) / (imgn_train.size()[0] * 2)
loss.backward()
optimizer.step()
`

这个代码是你在首页推荐的，http://www.ipol.im/pub/pre/231/ 里面的代码。
![image](https://user-images.githubusercontent.com/4082491/141054847-7854f43f-2440-4720-92aa-17ce4f1cbb88.png)

你看他计算 loss 的时候，是将模型的输出与 noise 进行计算，也就是说模型预测的是 噪声，怎么感觉跟你的论文说的不一样啊

**Comment by 7568** [comment]
没事了，稍微改一下就行了
